test/library/draft/Vector/these/vectorTheseParStandalone.chpl

```diff
@@ -14,10 +14,11 @@ forall x in vec1 do
 // Vector 2 has `parSafe=true`, here.
 forall x in vec1 with (ref vec2) do
   vec2.append(x);
-
 vec2.sort();
 
+/*
 for (x, y) in zip(vec1, vec2) do
   assert(x == y);
+  */
```

Comment:
What was wrong with this check?

Reply:
My fault

Reply:
https://github.com/chapel-lang/chapel/pull/16048/commits/5ab0d2c284226e75d3eb772bf38590ea1865e2a6